python/lsst/ts/eas/eas_csc.py

```diff
@@ -308,6 +347,74 @@ async def run_control(self) -> None:
                 )
                 break
 
+    async def m1m3_stop(self) -> None:
+        """Ends the M1M3 control loop.
+
+        Commands the mixing valve to close and the fans to stop.
+        Waits for telemetry to report that the mixing valve
+        has closed and the fans have stopped.
+        """
+
+        if self.simulation_mode != 0:
+            return
+
+        try:
+            await self.m1m3ts.cmd_setMixingValve.set_start(
+                mixingValveTarget=0, timeout=SAL_TIMEOUT
+            )
+            self.old_valve_position = 0
+
+            await self.m1m3ts.cmd_heaterFanDemand.set_start(
+                heaterPWM=[0] * 96,
+                fanRPM=[0] * 96,
+                timeout=SAL_TIMEOUT,
+            )
+
+            await asyncio.gather(self.mixing_value_wait(), self.fan_wait())
+        except Exception:
+            await self.fault(
+                code=THERMAL_LOOP_ERROR,
+                report="Failed to get mixing valve telemetry from M1M3TS.",
+                traceback=traceback.format_exc(),
+            )
+
+    async def mixing_valve_wait(self) -> None:
```

Comment:
Same ideas as above for the `m1m3_stop` method. Please, make sure the method name is more expressive.